cv.md

```diff
@@ -0,0 +1,29 @@
+# Sonya Bogolubova, QA engineer :computer:
+
+### About me
+Fast learner, team player, have good communication skills, responsible, disciplined, open-minded.
+Willing to improve professional skills and enhance knowledge base as well as professional experience across multiple disciplines. Have an aim to learn new specialties and extend abilities in programming and automation areas. 
+
+#### :e-mail: sonya.bogolubova@gmail.com
+#### :iphone: +375293601078
+
+### Core Skills
+Quality assurance of web based, desktop and mobile applications. Design of different types of documentation: test-cases, check lists, test plans, bug reports.
+
+### Extra Skills
```

Comment:
It's better to divide extra skills into some parts. Example: 
**Design tools**: Adobe Photoshop, Adobe Illustrator, Figma